lib/compiler/src/compiler.rs

```diff
@@ -79,6 +79,12 @@ where
 
 /// An implementation of a Compiler from parsed WebAssembly module to Compiled native code.
 pub trait Compiler: Send {
+    /// Returns a descriptive name for this compiler.
+    // TODO!: Probably want to make this mandatory on the next major bump.
+    fn name(&self) -> &str {
+        "UNKNOWN"
+    }
```

Comment:
We should just not have this implemented, even if it's a breaking change.
Right now I don't know of any user/company using their custom compilers

Reply:
done